package.json

```diff
@@ -147,56 +145,54 @@
     "@types/bunyan-format": "^0.2.5",
     "@types/compression": "^1.7.2",
     "@types/connect-flash": "0.0.37",
-    "@types/connect-redis": "^0.0.20",
```

Comment:
this and `@types/redis` not needed as the type defs are included in the libraries now